graphic_templates/waffle_chart/graphic_config.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+import base_filters
+
+COPY_GOOGLE_DOC_KEY = '1tFzuCif-4K9G10c3EpjJFp4FDs-fXj7-qUKKNsiBVxs' # ABC copy
```

Comment:
I've created an appropriate template from this based off the pie chart template. Please update this key to 1lNeDBYiAU3EqL9Dg5YKK3qK2BBnBq1_MAgmp5pKffW0